### ISSUE TITLE
chore: study move to Landing Zone Accelerator on AWS

### ISSUE THREAD

**Issue by flochaz** [awslabs/aws-bootstrap-kit#103]
AWS Solutions recently released a new solution close to what we have done here. It would be interesting to look into how to move to it as a base and only propose adapted config files.

Links:
* https://aws.amazon.com/fr/solutions/implementations/landing-zone-accelerator-on-aws/?did=sl_card&trk=sl_card
* https://github.com/awslabs/landing-zone-accelerator-on-aws

**Comment by moltar** [comment]
That solution is very "heavy-handed", and expensive for a small project.

Ideally, any solution should be able to be deployed in layers of complexity needed for each project.

I think anything that is just starting, ideally, should fit within a free tier, or a few dollars per month max.

While Landing Zone Accelerator is estimated at > $400/mo of just base costs.

**Comment by flochaz** [comment]
Agree, that is the reason why we would like to see how customizable it is before doing anything. If it has been built with proper feature flags in place, you may easily deactivate some of the services and make it evolve at your own pace.

**Comment by moltar** [comment]
Yup, that would excellent. I think having "tiers" with up-front pricing estimates of recommended settings (as a collection of flags) that one can opt-in to would be a great DX.

E.g.

1. MVP
2. Medium tier _(don't have a good name)_
3. Enterprise

**Comment by moltar** [comment]
Kind of like ESLint config bundles. Don't need to think about it too much. I think most people would be price-driven _or_ regulation-driven (compliance).
